vignettes/tagged_na_usage.Rmd

```diff
@@ -26,3 +43,13 @@ na_tag(x) # is used to read the tagged NA in most other functions they are still
 print_tagged_na(x) # Is used to print the na as well as their tag
 ```
 
+## CCHS approach to coding missing data
+`cchsflow` recodes category values as:
+
+| `tagged_na` | Category value | Label |
+|----------------|--------------|
+| NA(a) | 6                  | not applicable     |
+| NA(b) | 7                  | don't know         |
+| NA(b) | 8                  | refusal            |
+| NA(b) | 9                  | not stated         |
+| NA(b) | question not asked in the survey cycle  |
```

Comment:
Would you like variables/questions not asked to be tagged as NA(b)? Within the CCHS, variables that weren't asked to participants were coded as not applicable. For example, if a section of the CCHS was not asked to respondents from a particular province, those respondents would be coded as not applicable (NA(a)) as opposed to not stated (NA(b))

Reply:
Definitely should be NA(a)